sphinx_search/_static/css/rtd_sphinx_search.css

```diff
@@ -186,11 +183,13 @@
 }
 
 /* Path of each search result */
+
```

Comment:
These spaces are a bit strange but I don't have a very strong opinion about it.

Reply:
That was just to make the css a little cleaner.